### PR TITLE
Update aws.md

### PR DESCRIPTION
**Pull request by andrea-grapl**
AWS doc changes - tested through deployment.

### CODE REVIEW

**Review by colin-grapl — APPROVED** [review]

File: docs/setup/aws.md [hunk 1]
```diff
@@ -93,19 +93,21 @@ collection of these environment variables:
 
 ```bash
 # example values
-export GRAPL_DEPLOYMENT_NAME="some-grapl-deployment-name"
+export GRAPL_DEPLOYMENT_NAME="grapl-deployment-name"
+export DEPLOYMENT_NAME="grapl-deployment-name"
```
Comment (jgrillo-grapl) [inline docs/setup/aws.md:97]: I'd prefer to normalize these in code, if possible, rather than having two different variable names for the same thing.

**Review by jgrillo-grapl — CHANGES_REQUESTED** [review]

File: docs/setup/aws.md (hunk 1 above)
Comment (andrea-grapl) [reply docs/setup/aws.md:97]: Can we keep this for now and I'll open a new ticket to fix this, I don't want others to run into issues with deployments.

File: docs/setup/aws.md [hunk 2]
```diff
@@ -93,19 +93,21 @@ collection of these environment variables:
...
 export GRAPL_REGION="us-xxxx-n"
-export GRAPL_CDK_WATCHFUL_EMAIL="someone+watchful@example.com"
-export GRAPL_CDK_OPERATIONAL_ALARMS_EMAIL="someone+alarm@example.com"
-export GRAPL_CDK_SECURITY_ALARMS_EMAIL="someone+alarm@example.com"
+export GRAPL_CDK_WATCHFUL_EMAIL="email-for-watchful@example.com"
+export GRAPL_CDK_OPERATIONAL_ALARMS_EMAIL="email-for-op-alarm@example.com"
+export GRAPL_CDK_SECURITY_ALARMS_EMAIL="email-for-sec-alarm@example.com"
 export GRAPL_DGRAPH_INSTANCE_TYPE='xn.size' # e.g., 'i3.large'
 ```
+Note: Arguments for `GRAPL_DEPLOYMENT_NAME`and `DEPLOYMENT_NAME` should have the same deployment value.
```
Comment (ajm-grapl) [inline docs/setup/aws.md:106]: missing space after GRAPL_DEPLOYMENT_NAME

File: docs/setup/aws.md (hunk 1 above)
Comment (ajm-grapl) [reply docs/setup/aws.md:97]: Could we reference that GitHub issue from the docs, and just say the duplication is a workaround for now?

Comment (jgrillo-grapl) [reply docs/setup/aws.md:97]: https://github.com/grapl-security/grapl/pull/717

**Review by jgrillo-grapl — APPROVED** [review]
👍

**Pull request merged**